macos/Sources/Features/QuickTerminal/QuickTerminalSize.swift

```diff
@@ -0,0 +1,84 @@
+import GhosttyKit
+
+struct QuickTerminalSize {
```

Comment:
We should put this in the `Ghostty` folder because its a Swift structure wrapper around a libghostty type.